Workade/Extentions/Color+.swift

```diff
@@ -6,3 +6,31 @@
 //
 
 import Foundation
+import UIKit
+
+// MARK: Color
+// self.view.backgroundColor = .theme.primary
+
+extension UIColor {
+    static let theme = ColorTheme()
+    
+    static func rgb(_ rgbValue: Int) -> UIColor! {
```

Comment:
```suggestion
    static func rgb(_ rgbValue: Int) -> UIColor {
```
[P2] 강제 언래핑을 한 이유가 있을까요?

Reply:
그동안 프로젝트 하면서 항상 반복적으로 가져와서 쓰던 코드라 해당 이니셜라이저가 실패가능할 일이 없다는 것을 이제서야 알았네요.. 부주의한 습관이었던 것 같습니다. 좋은 지적 감사합니다!!